tests/main/views/test_frameworks.py

```diff
@@ -161,6 +167,18 @@ def test_create_fails_if_slug_is_invalid(self):
         assert response.status_code == 400
         assert json.loads(response.get_data(as_text=True))["error"] == "Invalid slug value 'this is/invalid'"
 
+    def test_create_fails_if_direct_award_and_further_competition_false(self):
+        framework = self.framework(slug="this is/invalid")
+        framework['frameworks']['hasDirectAward'] = False
+        framework['frameworks']['hasFurtherCompetition'] = False
+
+        response = self.client.post("/frameworks",
+                                    data=json.dumps(framework),
+                                    content_type="application/json")
+
+        assert response.status_code == 400
+        assert json.loads(response.get_data(as_text=True))["error"] == "Invalid slug value 'this is/invalid'"
```

Comment:
I think we're testing the wrong validation error here 😸

Reply:
😂 

Concentrate Sam!

Reply:
Fixed 👍